### PR TITLE
Adding initialization headers to a WebSocket GraphQL client

### PR DESCRIPTION
**Pull request by codewalkerdan**
Not just authorization, but other headers could be added that may be required when initializing a web socket connection.
It supports adding authorization tokens

### CODE REVIEW

File: src/GraphQL.Client.Abstractions.Websocket/GraphQLWebSocketRequest.cs [hunk 1]
```diff
@@ -34,9 +34,9 @@ public string Type
         /// <summary>
         /// The payload of the websocket request
         /// </summary>
-        public GraphQLRequest Payload
+        public object Payload
```
Comment (sungam3r) [inline src/GraphQL.Client.Abstractions.Websocket/GraphQLWebSocketRequest.cs:37]: breaking change

File: src/GraphQL.Primitives/GraphQL.Primitives.csproj [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <Description>GraphQL basic types</Description>
     <RootNamespace>GraphQL</RootNamespace>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Version>1.0.0</Version>
```
Comment (sungam3r) [inline src/GraphQL.Primitives/GraphQL.Primitives.csproj:9]: ?

File: src/GraphQL.Client/GraphQL.Client.csproj [hunk 3]
```diff
@@ -5,6 +5,14 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <RootNamespace>GraphQL.Client.Http</RootNamespace>
+    <Authors>Deinok,Alexander Rose,graphql-dotnet</Authors>
```
Comment (sungam3r) [inline src/GraphQL.Client/GraphQL.Client.csproj:8]: https://github.com/graphql-dotnet/graphql-client/blob/master/root.props

**Review by sungam3r — CHANGES_REQUESTED** [review]
These edits don't seem right to me.

**Comment by rose-a** [comment]
@codewalkerdan Thanks for your contribution. I'm closing this in favor of #294, which basically does the same thing albeit keeping types and names close to the [Apollo GraphQL over WebSocket Protocol Spec](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md#gql_connection_init), which this library aims to fully implement.

**Comment by codewalkerdan** [comment]
Sorry for not replying. I have little time, but thank you for the feedback.

**Pull request closed without merging**